modules/nf-core/motus/preplong/tests/main.nf.test

```diff
@@ -0,0 +1,75 @@
+nextflow_process {
+
+    name "Test Process MOTUS_PREPLONG"
+    script "../main.nf"
+    process "MOTUS_PREPLONG"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "motus"
+    tag "motus/downloaddb"
+    tag "motus/preplong"
+
+    setup {
+        run("MOTUS_DOWNLOADDB") {
+            script "modules/nf-core/motus/downloaddb/main.nf"
+            process {
+                """
+                input[0] = Channel.of([
+                    file('https://raw.githubusercontent.com/motu-tool/mOTUs/master/motus/downloadDB.py')
```

Comment:
This seems strange, why is the input a python script?

Reply:
In order to run the module, we need to run the download of the motus db first.This is how we have been doing with the motus profile as well. 

https://github.com/nf-core/modules/blob/master/modules/nf-core/motus/profile/tests/main.nf.test#L20

Reply:
But the script could just be part of the module?

Reply:
Could you please explain what you mean? We need the script to run the test.

Reply:
I think @SPPearce means we could just copy and paste teh code from the mOTUs repository and dump it straight into the module itself.

I think we did this @SPPearce as the script occasioanlly changes to account for changes in where the the database gets downloaded from or something, so it was easier to just download on the fly.

Reply:
Yes, that is what I meant, to include the script itself with the module for the download module.

Reply:
But if there is a reason behind it then that is fine. It isn't relevant for this module anyway.